acceptance-tests/src/main/java/io/blueocean/ath/BasePage.java

```diff
@@ -0,0 +1,93 @@
+package io.blueocean.ath;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Provides utility methods to downstream test classes
+ */
+public class BasePage {
```

Comment:
This should be named something else...

Reply:
Perhaps `BasePageObject` or `AbstractPageObject` and made abstract if not meant to be instantiated directly.